src/vsr.zig

```diff
@@ -957,7 +959,7 @@ pub fn sector_ceil(offset: u64) u64 {
 }
 
 pub fn checksum(source: []const u8) u128 {
-    var target: [32]u8 = undefined;
-    std.crypto.hash.Blake3.hash(source, target[0..], .{});
-    return @bitCast(u128, target[0..@sizeOf(u128)].*);
+    var target: u128 = undefined;
+    std.crypto.hash.Blake3.hash(source, std.mem.asBytes(&target), .{});
+    return target;
```

Comment:
Apologies @sentientwaffle let's rather keep this as is, since it was in fact safe as @ifreund clarified.

I think now with passing a `u128` we're actually passing too small a target, since Blake3 is a 256-bit hash.